docs/user/tutorials/install_epinio_cli.md

```diff
@@ -0,0 +1,56 @@
+#  Installation of the Epinio CLI
+
+If not done already, refer to [System Requirements](https://github.com/epinio/epinio#system-requirements).
+
+## Install Dependencies
+
+Follow these [steps](./install_dependencies.md) to install dependencies.
+
+## Install Epinio CLI
+
+### Download the Binary
+
+Find the latest version at [Releases](https://github.com/epinio/epinio/releases).
+Run one of the commands below, and replace \<epinio-version\> with the version of your choice, e.g. "v0.0.21".
+
+##### Linux
+
+```bash
+curl -o epinio -L https://github.com/epinio/epinio/releases/download/<epinio-version>/epinio-linux-amd64
+```
+
+##### MacOS
+
+```bash
+curl -o epinio -L https://github.com/epinio/epinio/releases/download/<epinio-version>/epinio-darwin-amd64
+```
+
+##### Windows
+
+```bash
+ curl -LO https://github.com/epinio/epinio/releases/download/<epinio-version>/epinio-windows-amd64.exe
```

Comment:
This is somewhat different in the options from the `curl` commands for linux and macos above.
Are these differences significant ?
If not why are they present ?
If yes, would the user need an explanation of why ?

Reply:
This part is copied from the existing documentation - it's identical to the current documentation, so no change. Assuming it was tested on Windows, and documented accordingly.